astropy/visualization/tests/test_norm.py

```diff
@@ -281,31 +286,55 @@ def test_invalid_stretch(self):
             simple_norm(DATA2, stretch="invalid")
 
 
+@pytest.mark.skipif(not HAS_PLT, reason="requires matplotlib.pyplot")
+@pytest.mark.parametrize("stretch", ["linear", "sqrt", "power", "log", "asinh", "sinh"])
+def test_simplenorm(stretch):
+    data = np.arange(25).reshape((5, 5))
+    snorm = SimpleNorm("linear", percent=99)
+    norm = snorm(data)
+    assert isinstance(norm, ImageNormalize)
+    assert_allclose(norm(data), simple_norm(data, "linear", percent=99)(data))
```

Comment:
whoops
```suggestion
    snorm = SimpleNorm(stretch, percent=99)
    norm = snorm(data)
    assert isinstance(norm, ImageNormalize)
    assert_allclose(norm(data), simple_norm(data, stretch, percent=99)(data))
```

Reply:
Fixed, thanks.  That's what I intended, but clearly I'm doing too many simultaneous tasks right now.

Reply:
no worries, we're all in this together 😄